src/OpenTelemetry/Logs/LogRecord.cs

```diff
@@ -156,10 +157,24 @@ public ActivityTraceFlags TraceFlags
         /// <summary>
         /// Gets or sets the log <see cref="Microsoft.Extensions.Logging.LogLevel"/>.
         /// </summary>
+        // todo: [Obsolete("Use Severity instead LogLevel will be removed in a future version.")]
         public LogLevel LogLevel
         {
-            get => this.ILoggerData.LogLevel;
-            set => this.ILoggerData.LogLevel = value;
+            get
+            {
+                if (!this.Data.Severity.HasValue)
+                {
+                    return LogLevel.Trace;
```

Comment:
Why Trace?

Reply:
Just to match the default behavior today. If you don't set `LogRecord.LogLevel` it is going to be automatically set to `default(LogLevel)` which is `Trace` (0).